### ISSUE TITLE
Buying TFTs with BTC on the Testnet

### ISSUE THREAD

**Issue by dmitri-alleo** [threefoldtech/test_feedback#125]
As it worked perfectly last week, today there is an issue when trying to buy TFTs on the Testnet

Once you specify the address at which you want to receive the TFTs it shows that the address is invalid, 
the same address is used as last time

![image](https://user-images.githubusercontent.com/65153287/167572013-46dd4461-5d82-4fe7-8f1a-75a1bb91f4a8.png)

**Comment by xmonader** [comment]
You can't use stellar mainnet addresses on stellar testnet.

**Comment by dmitri-alleo** [comment]
I'm using the address from my account on the Testnet Portal

**Comment by xmonader** [comment]
@dmitri-alleo please report the following: website URL, addresses used as plaintext.

**Comment by dmitri-alleo** [comment]
https://portal.test.grid.tf/account/5Gzh4sZieoqa2Wb9tmX7uRU7YfWQMkWmJgLXaD4Hv9KphY3k
GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4

![image](https://user-images.githubusercontent.com/65153287/167850216-849a69aa-86be-435b-9f87-22abee50da85.png)

![image](https://user-images.githubusercontent.com/65153287/167850310-5036d9af-9502-4002-9da4-4c2330f2082f.png)

**Comment by xmonader** [comment]
I meant the URL of the shop website you're using.

**Comment by dmitri-alleo** [comment]
It's https://test.gettft.com/gettft/

**Comment by sameh-farouk** [comment]
Hi, @dmitri-alleo this account can not be found on the stellar testnet network.

https://horizon-testnet.stellar.org/accounts/GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4

it could be a glitch on stellar test network, also per their docs a reset should be expected in such a test environment
> [the network is ephemeral, and resets periodically](https://developers.stellar.org/docs/glossary/testnet/#periodic-reset-of-testnet-data).

also

> A reset clears all ledger entries (such as accounts, trustlines, offers, etc), transactions, and historical data from both Stellar Core and Horizon, which is why developers should not rely on the persistence of any accounts or on the state of any balances when using testnet.

and

> The testnet resets once per quarter (every three months). The 2022 dates:
3/16/22
6/15/22
9/14/22
12/14/22

regardless of the reason for the disappearance of this account (I have one account of 3 testnet stellar accounts that have gone as well when I checked), we have nothing we can do about it.

**Comment by dmitri-alleo** [comment]
Thank you for the info @sameh-farouk , 
I've created the addresses/accounts back in February, and we've requested some TFTs from Threefold,
I suspect the funds were not the test Stellars, but the real Stellars bridged to TFTs, but somehow the funds can only be found on the Testnet portal, when I login with the same accounts to the Mainnet, the funds for similar addresses cannot be found.
But those are the funds which are used to provision all of the resources on the Testnet.
So, I'm not sure how it works.
I will try generating a new Stellar Testnet address separately from the Portal and will try using that one.
